### PR TITLE
feat: new util func to retrieve label definitions

### PR DESCRIPTION
**Pull request by shawnzhu**
## Design

The current convention comes from [Use Training Storage Path - SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/model-train-storage.html)

## Progress

* [x] new func
* [x] unit test
* [x] doc

### CODE REVIEW

**Comment by shawnzhu** [comment]
just realized it needs to cover the evaluation stage as well where the metadata dir is from env variable `METADATA_DIR`

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 repos:
--   repo: https://github.com/ambv/black
+  - repo: https://github.com/ambv/black
     rev: stable
     hooks:
-    - id: black
+      - id: black
 default_language_version:
-  python: python3.7
+  python: python3.11
```
Comment (epeters3) [inline .pre-commit-config.yaml:7]: I'd love your thoughts on this: should we wait to upgrade the Python version in another PR where we can do a full upgrade e.g. update the version of Python used in CI when the tests are run, and update the metadata in `setup.py` to reflect that version?

**Review by epeters3 — APPROVED** [review]
This utility looks great! I just left one comment about updating the Python version.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (epeters3) [reply .pre-commit-config.yaml:7]: Oh, nevermind! I just saw https://github.com/lifeomic/phc-sdk-py/pull/188, so looks like that has already been addressed.

**Pull request merged**